banano-quest/banano-quest/Protocols/Banano.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Banano.swift
+//  banano-quest
+//
+//  Created by Pabel Nunez Landestoy on 6/28/18.
+//  Copyright © 2018 Michael O'Rourke. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol Banano {
+    func refreshView()
+}
+
+class BananoQuestView: UIViewController, Banano {
```

Comment:
This should be called BananoQuestViewController, or change inheritance to UIView.